### ISSUE TITLE
Screen slanted

### ISSUE THREAD

**Issue by SANJU7667** [Genymobile/scrcpy#3924]
I had recently mirrored to my PC using this after few days I noticed that my devices is titled towards right a little how can I solve this issue. It also flicker sometimes. I am in fear that I broke my Android phone pls help

**Comment by rom1v** [comment]
> I had recently mirrored to my PC using this after few days I noticed that my devices is titled towards right

What do you mean? Could you post a screenshot?

**Comment by DKallx** [comment]
Yes! I came to research about this, it happens to me too... it's like this, I have a Xiaomi Poco F3 and I mirror its screen via USB, I play Call Of Duty Mobile at 120fps and I mirror it at 60fps (but the problem happens regardless of FPS) , the screen has "Screen Tearing" the broken frames do not come out in the recording, only on the device screen, and only while Scrcpy is running, limiting the game in the "Max" option (60fps) does not occur, however in ultra ( 120fps) the problem happens, here is the link of a video I recorded a few minutes ago, in previous versions of Scrcpy this never happened to me

https://youtu.be/u6HUqjHdGFE

**Comment by rom1v** [comment]
@DKallx oh it happens on the device only, so it is a problem on the device when the screen is captured (not in scrcpy).

Maybe `scrcpy --max-fps=60` might limit the problem?

**Comment by DKallx** [comment]
I understand, but even without limiting the fps this happens, I'm not a developer, I just came to report what happens to me
